packages/application-generic/src/usecases/get-feature-flag/get-is-execution-log-queue-enabled.use-case.ts

```diff
@@ -0,0 +1,22 @@
+import { Injectable } from '@nestjs/common';
+import { FeatureFlagsKeysEnum } from '@novu/shared';
+
+import { FeatureFlagCommand } from './get-feature-flag.command';
+import { GetFeatureFlag } from './get-feature-flag.use-case';
+
+@Injectable()
+export class GetIsExecutionLogQueueEnabled extends GetFeatureFlag {
+  async execute(featureFlagCommand: FeatureFlagCommand): Promise<boolean> {
+    const value = process.env.IS_API_EXECUTION_LOG_QUEUE_ENABLED;
+    const fallbackValue = true;
```

Comment:
shouldn't be fallback `false`?

Reply:
Well, I thought about it, isn't it dependent on what we want to have as default? I thought it should be to continue to use the queue. and then we use the FF from there. lmkwyt.

Reply:
I think it should use the queue by default, and we will turn it off explicitly. Later we can remove the FF once we finalize on the wanted solution.